### PR TITLE
Envoy 1.25.3 Dockerfile and currency integration

### PR DESCRIPTION
**Pull request by sumitd2**
## Checklist

- [X] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [X] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [X] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.4
+
+ENV ENVOY_PATCH=https://raw.githubusercontent.com/ppc64le/build-scripts/master/e/envoy/envoy_1.25.3.patch
+ENV PACKAGE_NAME=envoy
+ENV PACKAGE_VERSION=1.25.3
+ENV PACKAGE_URL=https://github.com/envoyproxy/envoy/
+ENV PYTHON_VERSION=3.10.2
+ENV wdir=/tmp
+
+#Build and install Kong
```
Comment (seth-priya) [inline e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile:10]: Envoy instead of Kong?

**Review by seth-priya — COMMENTED** [review]
Please correct typo, rest looks good, @nikhil-kalbande please check for currency

File: e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile (hunk 1 above)
Comment (sumitd2) [reply e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile]: done

File: e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.4
```
Comment (nikhil-kalbande) [inline e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile:1]: @sumitd2 - Any particular reason that we are using ubi 8.4?

File: e/envoy/build_info.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "package_name" : "envoy",
+    "github_url": "https://github.com/envoyproxy/envoy",
+    "version": "v1.25.3",
+    "package_dir": "e/envoy",
+    "default_branch": "main",
+    "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+    "build_script": "envoy_1.25.3_ubi8.4.sh",
```
Comment (nikhil-kalbande) [inline e/envoy/build_info.json:8]: Also @sumitd2  Can you make changes in build script which are aligned to currency need. Handling failures for build and test with different exit codes

Comment (sumitd2) [reply e/envoy/build_info.json:8]: Oh i forgot

File: e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile (hunk 2 above)
Comment (sumitd2) [reply e/envoy/Dockerfiles/1.25.3_ubi_8/Dockerfile:1]: yes, the customer wanted it

File: e/envoy/build_info.json (hunk 3 above)
Comment (sumitd2) [reply e/envoy/build_info.json:8]: done

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**